src/devices/video/ef9365.cpp

```diff
@@ -749,18 +707,16 @@ int ef9365_device::draw_vector(uint16_t start_x,uint16_t start_y,short delta_x,s
 //  from the charset.
 //-------------------------------------------------
 
-int ef9365_device::get_char_pix( unsigned char c, int x, int y )
+int ef9365_device::get_char_pix(uint8_t c, int x, int y)
 {
-	int char_base,char_pix;
-
-	if(c<96)
+	if (c < 96)
 	{
-		if( x < 5 && y < 8 )
+		if (x < 5 && y < 8)
 		{
-			char_base =  c * 5;
-			char_pix = ( y * 5 ) + x;
+			uint32_t char_base = c * 5;
+			uint32_t char_pix = (y * 5) + x;
 
-			if ( m_charset[char_base + (char_pix>>3)] & ( 0x80 >> (char_pix&7)) )
+			if (BIT(m_charset[char_base + (char_pix >> 3)], 7 - (char_pix & 7)))
```

Comment:
You can simplify `7 - (char_pix & 7)` to `~char_pix & 7` here (it has less 7 that way).

Reply:
Applied.